ChangeLog.md

```diff
@@ -1600,3 +1600,18 @@ with the same id.
 ### Leiningen 2
 
 Elastisch now uses [Leiningen 2](https://github.com/technomancy/leiningen/wiki/Upgrading).
+
+### elastisch.conversion
```

Comment:
Please take a look at how the change log is structured. We do not list changes by module and they come in reverse chronological order.